docs/cookbook/payments/custom-invoice.rst

```diff
@@ -36,17 +36,31 @@ Example custom configuration:
 
 Make sure to clear the cache each time the configuration is changed.
 
-If you have a permission issue when generating an invoice, you may also need to update the ``config/packages/knp_snappy.yaml`` configuration file:
+.. image:: ../../_images/cookbook/custom-invoice/pdf_with_custom_logo.png
 
-.. code-block:: yaml
+How to add more graphics to the Invoice?
+--------------------------------------------
+In case you would like to add an extra graphic to your Invoice twig template, it is super important to provide access to this file. Let's say you would like to add second image to the Invoice. You may face the problem that ``wkhtmltopdf`` from version 0.16 disables access to the local files by default. Fortunately, there are two options to deal with it:
```

Comment:
Split this line into several lines and there should be 0.12.6 version I guess 🤔

Reply:
Yep, should be 0.12.6 ;d Thanks for catching it.